controllers/performanceprofile_controller.go

```diff
@@ -18,16 +18,17 @@ package controllers
 
 import (
 	"context"
+	e "errors"
+	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components/kubeletconfig"
+	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components/runtimeclass"
+	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components/tuned"
```

Comment:
please move back these imports below, because it breaks proper grouping:

1. stdlib
2. third party
3. our own modules